eng/scripts/Merge-Coverage-Reports.ps1

```diff
@@ -0,0 +1,28 @@
+# Wrapper Script for ChangeLog Verification
+param (
+  [String]$RepoRoot,
+  [String]$BinDirectory
+)
+
+$CoverageFiles = Get-ChildItem "${RepoRoot}*coverage.cobertura.xml" -Recurse -File
+$CoverageFilesString
+
+ForEach ($file in $CoverageFiles)
+{
+    if ($CoverageFilesString)
+    {
+        $CoverageFilesString += ';'
+    }
+    $CoverageFilesString += $file.FullName
+}
+
+#Download ReportGenerator Tool from Nuget
+Invoke-WebRequest -MaximumRetryCount 10 -Uri "https://www.nuget.org/api/v2/package/ReportGenerator/4.6.4" `
```

Comment:
Can we not do a dotnet install of this tool instead of manually downloading it?

Reply:
dotnet tool install does not work, I guess its not a proper tools package.

Reply:
Looking at the site https://github.com/danielpalme/ReportGenerator there appears to be other consumption models like an AzureDevOps Task https://marketplace.visualstudio.com/items?itemName=Palmmedia.reportgenerator or a dotnet tool https://www.nuget.org/packages/dotnet-reportgenerator-globaltool. We should use one of those options over downloading this manually.

Reply:
I switched to using the pipeline task